code/src/cljs/sixsq/nuvla/ui/profile/events.cljs

```diff
@@ -136,7 +136,6 @@
                            ::spec/loading (disj loading :customer))}
             customer (assoc :fx [(when (general-utils/can-operation? "get-subscription" customer)
                                    [:dispatch [::get-subscription]])
-                                 [:dispatch [::close-modal]]
```

Comment:
You removed close modal from set-customer to fix the issue. But did it have an impact on other things like create subscription in modal or add-payment-method?

Reply:
I think moved the close modal event to the places dispatching `::set-customer` but now I'm not so sure anymore.

I'll check

Reply:
I missed two, see lines 279 and 295, those should be all I think.